.github/workflows/cromwell-test-run.yml

```diff
@@ -1,32 +1,56 @@
 name: Cromwell Test Run
 
 on:
+  pull_request:
   workflow_dispatch:
 
 jobs:
+  generate-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Find WDL Directories
+        id: set-matrix
+        run: |
+          # Find all directories containing .wdl files
+          dirs=$(find . -type f -name "*.wdl" -exec dirname {} \; | sort -u | sed 's|^\./||' | jq -R . | jq -s | jq -c .)
+          echo "Directories: $dirs"
+          echo "matrix=$dirs" >> $GITHUB_OUTPUT
+
   testrun:
+    needs: generate-matrix
```

Comment:
Looks like this is missing from the womtools yml, might be what's causing the issue..